src/server/json_family_test.cc

```diff
@@ -515,15 +515,14 @@ TEST_F(JsonFamilyTest, ObjLenLegacy) {
   ASSERT_THAT(resp, "OK");
 
   /* Test simple response from only one value */
-
   resp = Run({"JSON.STRLEN", "json"});
   EXPECT_THAT(resp, ErrArg("wrong JSON type of path value"));
 
   resp = Run({"JSON.OBJLEN", "json", ".a"});
   EXPECT_THAT(resp, IntArg(0));
 
   resp = Run({"JSON.OBJLEN", "json", ".a.*"});
-  EXPECT_THAT(resp, ErrArg("wrong JSON type of path value"));
+  EXPECT_THAT(resp, ArgType(RespExpr::NIL));
```

Comment:
In Redis I'm getting WRONGTYPE error:
```
127.0.0.1:6379> json.set json . '{"a":{"b":"c"}}'
OK
127.0.0.1:6379> json.objlen json .a.*
(error) WRONGTYPE wrong type of path value - expected object but found string
```

Reply:
Same with this PR but it depends if there was a match or not. With this test the input is:
`{"a":{}, "b":{"a":"a"}, "c":{"a":"a", "b":"bb"}, "d":{"a":1, "b":"b", "c":{"a":3,"b":4}}, "e":1}` and here there are 0 matches, hence NIL